video/out/d3d11/context.c

```diff
@@ -563,6 +577,9 @@ static bool d3d11_init(struct ra_ctx *ctx)
     if (!mp_d3d11_create_swapchain(p->device, ctx->log, &scopts, &p->swapchain))
         goto error;
 
+    if (ctx->opts.composition) {
```

Comment:
nitpick, but please be consistent about brackets, we don't use them for single line statements.